README.md

```diff
@@ -0,0 +1,3 @@
+# Aparat downloader
+
+This is a puppeteer bot with Nodejs that download playlist from [Aparat](https://www.aparat.com/).
```

Comment:
A Node.js project that uses puppeteer to download playlists from [Aparat](https://www.aparat.com/).